lib/tasks/publishing_api.rake

```diff
@@ -56,19 +56,22 @@ namespace :publishing_api do
       $stderr.puts "Class: #{item.class}; id: #{item.id}; Error: #{err.message}"
     end
 
-    class_name = args[:item_class]
-    if class_name == 'Edition'
-      klass = Edition.published
-    else
-      klass = class_name.constantize
-    end
+    args[:item_classes].split(',').each do |class_name|
+      if class_name == 'Edition'
+        editions = Edition.published
+      else
+        editions = class_name.constantize.published
```

Comment:
Checked this, and the only thing on our list that doesn't have a `published` scope is `Policy`.